pyproject.toml

```diff
@@ -102,7 +104,7 @@ warn_redundant_casts = true
 warn_unreachable = true
 warn_unused_configs = true
 warn_unused_ignores = true
-exclude = "^build/|^docs/"
```

Comment:
could we revert this change?

Reply:
It's reverted (save for the addition of `^temp/`) as part of the sensAI dependency commit.